dev/io.openliberty.org.jboss.resteasy.cdi.ee10/resources/META-INF/services/jakarta.enterprise.inject.spi.Extension

```diff
@@ -1 +0,0 @@
-io.openliberty.org.jboss.resteasy.common.component.LibertyResteasyCdiExtension
```

Comment:
These seems wrong to delete.  I figured the other one would be the one that would be deleted.

Reply:
The entire project is NOT removed from what I can see.  This class is in this project so it would seem strange to remove it from here.  Adam removed the duplication of this file in the common.ee10 project.